packages/tradershub/src/helpers/displayMoney.ts

```diff
@@ -0,0 +1,25 @@
+import { useAuthorize } from '@deriv/api';
+
+type TCurrency = NonNullable<ReturnType<typeof useAuthorize>['data']['currency']>;
+type TPreferredLanguage = ReturnType<typeof useAuthorize>['data']['preferred_language'];
+
+export const displayMoney = (
```

Comment:
add docs

Reply:
as you implemented this, if you add comments will be better here and then just move to utils

Reply:
fine !